server/openapi.go

```diff
@@ -51,16 +58,43 @@ func (s *Server) sqlCheckController(c echo.Context) error {
 		return echo.NewHTTPError(http.StatusBadRequest, "Missing required SQL statement")
 	}
 
-	dbType := c.QueryParams().Get("databaseType")
-	if dbType == "" {
-		return echo.NewHTTPError(http.StatusBadRequest, "Missing required database type")
+	ctx := c.Request().Context()
+	databaseID := c.QueryParams().Get("databaseID")
+
+	var dbType db.Type
+	var catalog catalog.Catalog = &catalogService{}
+
+	if databaseID != "" {
```

Comment:
I thought we plan to change to database name?

Reply:
If so we also need to let users pass the instance id to ensure uniqueness. Are we okay with this? (seems it's not easier for users as they still need to find the id on bytebase)